scripts/integer-tests.sh

```diff
@@ -24,35 +24,64 @@ else
     n_threads=6
 fi
 
-# block pbs are too slow for high params
-# mul_crt_4_4 is extremely flaky (~80% failure)
-# test_wopbs_bivariate_crt_wopbs_param_message generate tables that are too big at the moment
-# same for test_wopbs_bivariate_radix_wopbs_param_message_4_carry_4
-# test_integer_smart_mul_param_message_4_carry_4 is too slow
-filter_expression=''\
+if [[ "${BIG_TESTS_INSTANCE}" == FALSE ]]; then
```

Comment:
this needs to be `!= TRUE` otherwise just launching the script outside the makefile will fail